### PR TITLE
Removed duplicate question in git-quiz.md

### PR DESCRIPTION
**Pull request by eddiestefanescu**
## PR Checklist

This PR is ready for review and meets the requirements set out
in [Suggestion how to contribute](CONTRIBUTING.md)

### DOD

- [ ] I have added new quiz{'s}
- [ ] I have added new reference link{'s}
- [x] I have made small correction/improvements

### Changes / Instructions

_Add instructions to me, please type here, thanks_

### CODE REVIEW

**Comment by eddiestefanescu** [comment]
I removed Question 133 which was a duplicate of Question 26 (which has been corrected previously). In addition to being a duplicate, Q133 also was showing the wrong answer, making it very confusing for users.

File: git/git-quiz.md [hunk 1]
```diff
@@ -1030,7 +1030,11 @@ Untracked files:
 - [ ] `git clean`
 - [ ] `git remove .`
 
+<<<<<<< HEAD
```
Comment (Ebazhanov) [inline git/git-quiz.md:1033]:
```suggestion
<<<<<<< HEAD
```
please remove

File: git/git-quiz.md [hunk 2]
```diff
@@ -1030,7 +1030,11 @@ Untracked files:
...
+Note: broccoli is an untracked file and will require git clean -f to be removed. On the other hand "git clean" by itself (without -f) may result in an error like this: *git clean fatal: clean.requireForce defaults to true and neither -i, -n, nor -f given; refusing to clean*
+=======
```
Comment (Ebazhanov) [inline git/git-quiz.md:1035]:
```suggestion

```

File: git/git-quiz.md [hunk 3]
```diff
@@ -1030,7 +1030,11 @@ Untracked files:
...
 Note: broccoli is an untracked file and will require git clean -f to be removed. On the other hand "git clean" by itself (without -f) may result in an error like this: _git clean fatal: clean.requireForce defaults to true and neither -i, -n, nor -f given; refusing to clean_
+>>>>>>> eabf07401dca2192893b1880e838ef587e743801
```
Comment (Ebazhanov) [inline git/git-quiz.md:1037]:
```suggestion
```

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
pls do clean up before we merge, thanks

**Comment by eddiestefanescu** [comment]
I got it... it's long overdue. I'll take care of it this weekend. I'll close the PR for now and resubmit it later.

**Pull request closed without merging**